src/Utils/Responsive/Media.tsx

```diff
@@ -0,0 +1,162 @@
+import * as theme from "@artsy/palette"
+import React from "react"
+import styled, { css, InterpolationValue } from "styled-components"
+
+/**
+ * TODO:
+ *
+ * Something like this doesn’t work, because an attribute can only be used once
+ * in a media query, presumably the last one wins:
+ *
+ *   <Match not any={[breakpoints.xs, breakpoints.sm, breakpoints.md]} />
+ *
+ * Generates:
+ *
+ *   @media not all and (max-width: 767px), not all and (max-width: 899px) and (min-width: 768px), not all and (max-width: 1023px) and (min-width: 900px)
+ *
+ * To overcome this we could define these attributes as actual props and then introduce a range matcher. E.g.
+ *
+ *   breakpoints.xs = Query.create({ maxWidth: 767 })
+ *   breakpoints.md = Query.create({ minWidth: 900, maxWidth: 1023 })
+ *   <Match not from={breakpoints.xs} to={breakpoints.md} />
+ *
+ * Which would then generate:
+ *
+ *   @media not all (max-width: 1023px)
+ */
```

Comment:
Could also go with something like https://github.com/morajabi/styled-media-query (https://github.com/morajabi/styled-media-query/blob/master/src/index.js)

Reply:
Oh, that's pretty awesome.

Reply:
Nice, I like this pattern. I’m going to make another example that uses this instead.